src/fu-util.c

```diff
@@ -1376,10 +1376,28 @@ fu_util_get_updates (FuUtilPrivate *priv, gchar **values, GError **error)
 	if (!fu_util_perhaps_refresh_remotes (priv, error))
 		return FALSE;
 
-	/* get devices from daemon */
-	devices = fwupd_client_get_devices (priv->client, NULL, error);
-	if (devices == NULL)
+	/* handle both forms */
+	if (g_strv_length (values) == 0) {
+		/* activate anything with _NEEDS_ACTIVATION */
```

Comment:
this comment seems wrong

Reply:
Sorry, forgot to read after copy/paste :( Thanks for catching